scripts/qa_crowbarsetup.sh

```diff
@@ -3486,7 +3489,7 @@ function onadmin_testsetup()
 {
     pre_hook $FUNCNAME
 
-    if iscloudver 5plus; then
+    if [ "$want_multidnstest" = 1 ] && iscloudver 5plus; then
```

Comment:
how about simply checking that the dns proposal has multiple server roles applied? that would make it work in both cases (batch and custom config).